srml/balances/README.adoc

```diff
@@ -0,0 +1,90 @@
+# Balances Module
+
+The balances module provides the functionality for handling balances.
+
+## Overview
+
+The balances module provides functions for:
+
+- Getting and setting free balance
+- Retrieving total, reserved, and unreserved balances
+- Repatriating a reserved balance to a beneficiary account that exists
+- Transfering a balance between accounts (when not reserved)
+- Slashing an account balance
+- Account removal
+- Lookup of an index to reclaim an account
+- Increasing or decreasing total stake
+- Setting and removing locks on chains that implement `LockableCurrency`
+
+The dispatchable function `transfer` ensures that the sender has signed the transaction. When using the publicly exposed functions in the implementation, you will need to do these checks in your runtime, as many functions will affect storage without ensuring, for example, that the sender is the signer.
+
+## Public Interface
+
+### Types
+
+- Balance
+- AccountId
+
+### Dispatchable Functions
+
+// TODO: Add link to rust docs (https://github.com/paritytech/substrate-developer-hub/issues/24)
+- `transfer` - Transfer some liquid free balance to another staker.
+- `set_balance` - Set the balances of a given account. Only dispatchable by a user with root priviledges.
+
+## Usage
+
+The following example shows how to use the balances module in your custom module.
+
+1 Import the `balances` module and derive your module configuration trait with the balances trait.
+
+```
```

Comment:
might as well use Rust syntax highlighting with `rust` after the first three dashes that initiate the code block (throughout the doc)
```rust
example::highlighted!()
```

Reply:
Didn't realize adoc formatted for rust, will update in next commit.